app/jobs/regular/problem_check.rb

```diff
@@ -13,17 +13,23 @@ def execute(args)
       retry_count = args[:retry_count].to_i
       identifier = args[:check_identifier].to_sym
 
-      check = AdminDashboardData.problem_scheduled_check_klasses[identifier]
+      check = ::ProblemCheck[identifier]
```

Comment:
I want to rename the jobs (e.g. `Jobs::RunProblemCheck`) to disambiguate the class names.

Reply:
I was thinking perhaps we should warn, too.